spec/acceptance/support/puppetdb.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+def has_puppetdb
+  case host_inventory['facter']['os']['name']
+  when 'Debian'
+    return false if ENV['BEAKER_PUPPET_COLLECTION'] == 'puppet6' && host_inventory['facter']['os']['release']['major'] == '11'
+  end
+
+  true
```

Comment:
You can consider this style as well.
```suggestion
    ENV['BEAKER_PUPPET_COLLECTION'] != 'puppet6' && host_inventory['facter']['os']['release']['major'] != '11'
  else
    true
  end
```

Reply:
The `&&` should also be changed to `||` but it feels less readable in my opinion so for now I will merge as is so that other PR can benefit from the fixes for Debian / Ubuntu.